app/server/src/main/scala/org/bitcoins/server/ServerJsonModels.scala

```diff
@@ -1598,4 +1618,47 @@ trait ServerJsonModels {
     js.arr.foldLeft(Vector.empty[OracleAttestmentTLV])((vec, tlv) =>
       vec :+ jsToOracleAttestmentTLV(tlv))
   }
+
+  def jsToAESPassword(js: Value): Option[AesPassword] = {
+    js match {
+      case Str(str) =>
+        Some(AesPassword.fromString(str))
+      case Null =>
+        None
+      case Arr(_) | False | True | Num(_) | Obj(_) =>
+        throw new IllegalArgumentException("password must be a string or null")
+    }
+  }
+
+  def jsToStringOpt(js: Value): Option[String] = {
+    js match {
+      case Str(str) =>
+        Some(str)
+      case Null =>
+        None
+      case Arr(_) | False | True | Num(_) | Obj(_) =>
+        throw new IllegalArgumentException("password must be a string or null")
+    }
+  }
+
+  def jsToWalletNameAndPassword(
+      js: Value): (Option[String], Option[AesPassword]) = {
+    js match {
+      case Arr(arr) =>
+        arr.toList match {
+          case walletNameJs :: passJs :: Nil =>
```

Comment:
This means the wallet name has to be defined to decrypt. I have a wallet locally that does not have a wallet name, but _does_ have a password.